test/integration/features/addon-xpaas.feature

```diff
@@ -19,7 +19,8 @@ which are then available in OpenShift to the user.
   Scenario Outline: User deploys, checks out and deletes several templates from XpaaS imagestream
    Given Minishift has state "Running"
     When executing "oc new-app <template-name>" succeeds
-     And services "<service-name>" rollout successfully
+     And executing "oc set probe dc/<service-name> --readiness --get-url=http://:8080<http-endpoint>" succeeds
```

Comment:
👍 Does this help to pass the test on the slow machine?

Reply:
@hferentschik Unfortunately not, it now also started to fail on centos-ci.

Reply:
@agajdosi I am confused by your comment, do you mean manually readiness probe is working fine but not in integration tests?

Reply:
@LalatenduMohanty TBH I can't tell right now. One possibility is that integration tests plays a role, another is that I was just unlucky when trying to replicate, third is that the it has something to do with network and time of the day.

The script I use:
```
oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git
oc expose svc/ruby-ex
oc set probe dc/ruby-ex --readiness --get-url=http://:8080
oc rollout status deploymentconfig ruby-ex --watch
curl http://ruby-ex-myproject.$(minishift ip).nip.io
```

will try it today once more and also on Windows.